rcsb/db/helpers/DocumentDefinitionHelper.py

```diff
@@ -220,6 +235,32 @@ def getSubCategoryAggregateFeatures(self, collectionName):
             logger.debug("Collection %s failing with %s", collectionName, str(e))
         return ret
 
+    def getIterableAttributeMetadata(self, collectionName, catName, atName):
+        """
+        Get metadata for iterable attributes (attribute types - scsv, csv, vbsv)
+        Example:
+
+            collection_iterable_attribute_metadata:
+                drugbank_core:
+                    - ATTRIBUTE_NAME: drugbank_info.drug_groups
+                    MIN_ITEMS: 1
+                    UNIQUE_ITEMS: TRUE
+        """
+        pD = {}
+        try:
+            # get atrribute metadata --
+            for clName, tDL in self.__cfgD["collection_iterable_attribute_metadata"].items():
+                if clName == collectionName:
+                    for tD in tDL:
+                        ff = str(tD["ATTRIBUTE_NAME"]).split(".")
+                        mValue = tD["MIN_ITEMS"]
+                        uValue = tD["UNIQUE_ITEMS"]
+                        if ff[0] == catName and ff[1] == atName:
+                            pD = {"minItems": mValue, "uniqueItems": uValue}
```

Comment:
Should/can the `mValue` and `uValue` assignments be nested under the `if ff[0]...` condition? E.g.:

```python
                        if ff[0] == catName and ff[1] == atName:
                            mValue = tD["MIN_ITEMS"]
                            uValue = tD["UNIQUE_ITEMS"]
                            pD = {"minItems": mValue, "uniqueItems": uValue}
```